iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/operator/OperatorContext.java

```diff
@@ -90,8 +96,8 @@ public Duration getMaxRunTime() {
     return maxRunTime;
   }
 
-  public void setMaxRunTime(Duration maxRunTime) {
-    this.maxRunTime = maxRunTime;
+  public static void setMaxRunTime(Duration maxRunTime) {
+    OperatorContext.maxRunTime = maxRunTime;
```

Comment:
why not remove this method since its nearly only used in tests and ClientRpcServiceImpl? I think there's no need to call this method both in tests and ClientRpcServiceImpl, too.
<img width="1087" alt="截屏2023-11-18 10 37 28" src="https://github.com/apache/iotdb/assets/108499334/c885533b-a3d3-4a70-b527-74e07c8aec10">

Reply:
Actually, I wanna keep this method for future usage like we want to change the time slice for running DN without restarting it.

Reply:
If I'm not mistaken, changing the time slice length without restarting should be done through calling a Session interface or executing a SQL . However, setting OperatorMaxRuntime doesn't seem to work, and it seems that what should be changed is the value in the Config, right?

Reply:
Yes, but in current situation, each operator's max run time is got from OperatorContext. And calling a Session interface or executing a SQL is to change the value in OperatorContext.

Reply:
I see.